### ISSUE TITLE
Midterm Peer Review 

### ISSUE THREAD

**Issue by rt359** [dd483/civilian-officer-outcomes#8]
Things I like: 
1)	The data preprocessing section is easy to follow and the actions performed align with the project goal and don’t significantly affect the outcome. 
2)	The histograms provided are easy to read and paint a clear picture of the overall look and shape of the data-for example, it is easy to see that gun point display and firearm are the most common force type value, which is different for all other race groups.  
3)	I like that the modeling section takes into account the possibility of overfitting through adding two regularizes, and considers various loss functions like OvA and multinomial. 
Things to Improve: 
1)	It would be helpful to add more visualizations that more clearly present the error findings in the preliminary model fitting. 
2)	The visualization in section 4 are slightly small and hard to read.
3)	In the future plans section, it would be helpful to elaborate on what potential false positive and false negative implications would be and how the group plans to explore the implications of both.
